### ISSUE TITLE
Test case 1A: Comments in observation.valueString should be in observation.note ? 

### ISSUE THREAD

**Issue by AlexisVZ-MDS** [hl7-be/lab#215]
Wouldn't it be better to just have "Positief" in valueString and "Identificatie van tegen ..." as comment in note ?

![image](https://github.com/hl7-be/lab/assets/93183816/89d03b64-8500-4fe9-b2ce-534b5cf36bb4)

**Comment by bdc-ehealth** [comment]
@AlexisVZ-MDS ,

Yes Alexis, that would be better. It would be even better to use a code for positive and negative (SNOMED CT 10828004/260385009). But this lab chose to use a string, and there is not a lot you can check on a string. 

Do you want to discuss this in the group?

**Comment by AlexisVZ-MDS** [comment]
Yes Bart, I think it is worth discussing in a group ! Thx for the quick follow-up :-).
